packages/web/pages/hustlers/mint-success.tsx

```diff
@@ -188,25 +189,40 @@ const AlertContainer = styled.div`
   animation: appear 0.1s linear 3.1s 1 forwards;
 `;
 
+const ScreenSaver = styled.div<{ image: string }>`
+  position: absolute;
+  top: 0px;
+  left: 0px;
+  width: 100%;
+  height: 100%;
+  z-index: 0;
+  background: ${({ image }) =>
+    `#000000 url('/images/hustler/${image}') center / cover no-repeat fixed;`};
+`;
+
 const MintSuccess = () => {
+  const [hasTransfered, setHasTransfered] = useState(false);
+  const hustler = useHustler();
   const [gangstaParty, setGangstaParty] = useState(false);
 
   const image = gangstaParty ? 'bridge_with_hustlers.png' : 'bridge_no_hustlers.png';
 
-  const ScreenSaver = styled.div`
-    position: absolute;
-    top: 0px;
-    left: 0px;
-    width: 100%;
-    height: 100%;
-    z-index: 0;
-    background: #000000 url('/images/hustler/${image}') center / cover no-repeat fixed;
-  `;
+  const listener = (block: string) => {
+    console.log(block);
+    setHasTransfered(true);
+  };
+
+  useEffect(() => {
+    hustler.on('TransferSingle', listener);
```

Comment:
this needs to be scoped to the person minting the token otherwise everyone will get everyone elses events

Reply:
Not sure how to implement this tbh, can you take a look at: https://github.com/ethers-io/ethers.js/issues/492#issuecomment-483776240, is this what you suggest?

Reply:
yeah something like that. i believe typechain provides some methods for creating the event filter

Reply:
Alright, will take a look